src/lib/utils/useIdentifyUser.ts

```diff
@@ -0,0 +1,15 @@
+import * as Sentry from "@sentry/react-native"
+import { GlobalStore } from "lib/store/GlobalStore"
+import { useEffect } from "react"
+import { SegmentTrackingProvider } from "./track/SegmentTrackingProvider"
+
+export const useIdentifyUser = () => {
```

Comment:
What do you think about adding a comment to document the function?

```suggestion
/**
 * bla bla blub
 */
export const useIdentifyUser = () => {
```

Reply:
oh yea good idea. I'll add it.